package.json

```diff
@@ -10,21 +11,19 @@
     "url": "https://github.com/DjangoGirls/djangogirls/issues"
   },
   "homepage": "https://github.com/DjangoGirls/djangogirls#readme",
-  "dependencies": {},
   "devDependencies": {
     "bootstrap": "^3.3.6",
-    "gulp": "^3.9.1",
-    "gulp-clean": "^0.3.2",
+    "del": "^6.0.0",
+    "fancy-log": "^1.3.3",
+    "gulp": "^4.0.2",
     "gulp-concat": "^2.6.0",
-    "gulp-environments": "^0.1.1",
+    "gulp-environments": "0.1.3",
     "gulp-rev": "^7.0.0",
-    "gulp-rev-css-url": "^0.1.0",
-    "gulp-rev-napkin": "^0.1.0",
```

Comment:
Semver works funny with `^0.x.y` (you can have an in-range breaking version, see [this](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004)), so I pinned these to an exact version.